Jenkinsfile

```diff
@@ -0,0 +1,105 @@
+def user_id
+def group_id
+node {
+    user_id = sh(returnStdout: true, script: 'id -u').trim()
+    group_id = sh(returnStdout: true, script: 'id -g').trim()
+}
+
+pipeline {
+    agent {
+        dockerfile {
+            dir '.ops/build-docker'
+            args '-v /var/run/docker.sock:/var/run/docker.sock -v /etc/group:/etc/group'
+            additionalBuildArgs "--build-arg USER_ID=${user_id} --build-arg GROUP_ID=${group_id}"
+        }
+    }
+    parameters {
+        booleanParam(name: 'CREATE_RELEASE_ZIP', defaultValue: false, description: 'Create the release zip from the dist directory')
+        string(name: 'AWS_INSTANCE_TYPE', defaultValue: 'm4.xlarge', description: 'AWS instance type for the databricks, chaos, or release tests (m4.4xlarge or larger is recommended if dataset is >1m)')
+        string(name: 'AWS_INSTANCE_CIDR', defaultValue: '127.0.0.1/32', description: 'The CIDR that will have access to Stardog running on 5820 in the environment for databricks, chaos, or release tests (if applicable)')
+        string(name: 'ARTIFACT_TAG', defaultValue: '', description: 'The tag used to stage artifacts (requires creating release zip and will push artifacts to artifactory, docker, etc.)')
+        string(name: 'SLACK', defaultValue: '', description: 'The slack channel to notify of the builds results.  This is typically #notifications-eng-gh.')
+    }
+    options {
+        timeout(time: 30, unit: 'MINUTES')
+    }
+    environment {
+        jenkinsOsxPub = credentials('jenkins.osx.pub')
+        jenkinsOsxPriv = credentials('jenkins.osx.priv')
+        artifactoryUsername = credentials('artifactoryUsername')
+        artifactoryPassword = credentials('artifactoryPassword')
+        AWS_ACCESS_KEY_ID = credentials('AWS_ACCESS_KEY_ID')
+        AWS_SECRET_ACCESS_KEY = credentials('AWS_SECRET_ACCESS_KEY')
+        STARDOG_GH_CREDS = credentials('stardog-ops-github-jenkins')
+        STARDOG_LICENSE = credentials('stardog_license_base64')
+        TF_VAR_artifactory_url = 'http://stardog.jfrog.io/stardog/stardog-releases-local'
+        TF_VAR_snapshot_url = 'http://stardog.jfrog.io/stardog/stardog-snapshots-local'
+        TF_VAR_artifactory_username = credentials('artifactoryUsername')
+        TF_VAR_artifactory_password = credentials('artifactoryPassword')
+        STARDOG_GITHUB_API_TOKEN = credentials('stardog-github-api-token')
```

Comment:
I'm guessing this is just copy/pasta, but it's not clear we need these. we can probably just add the ones you are using.

Reply:
I have removed them.